snuba/datasets/entities/sessions.py

```diff
@@ -44,18 +47,16 @@ def __init__(self) -> None:
                         ColumnToCurriedFunction(
                             None,
                             "duration_quantiles",
-                            FunctionCall(
-                                None,
-                                "quantilesIfMerge",
-                                (Literal(None, 0.5), Literal(None, 0.9)),
-                            ),
+                            FunctionCall(None, "quantilesIfMerge", quantiles,),
                             (Column(None, None, "duration_quantiles"),),
                         ),
-                        function_rule("sessions", "countIfMerge"),
-                        function_rule("sessions_crashed", "countIfMerge"),
-                        function_rule("sessions_abnormal", "countIfMerge"),
+                        function_rule("duration_avg", "avgIfMerge"),
+                        function_rule("sessions", "sumIfMerge"),
+                        function_rule("sessions_crashed", "sumIfMerge"),
+                        function_rule("sessions_abnormal", "sumIfMerge"),
+                        function_rule("sessions_errored_sum", "sumIfMerge"),
+                        function_rule("sessions_errored_uniq", "uniqIfMerge"),
```

Comment:
Is there a way in this query to do a `sessions_errored_sum + sessions_errored_uniq AS sessions_errored`?

Reply:
You need to do that in query processing.
if you want to use the same column name you need to change the translator from the logical query to the physical query (https://github.com/getsentry/snuba/blob/master/snuba/datasets/entities/sessions.py#L58). Now the translator transform the column `sessions_errored` in the query into the expression `uniqIfMerge(sessions_errored)`. It must become something like 
```
FunctionCall(alias, "sum", (
    FunctionCall(None, "uniqIfMerged", (Column(None, None, "sessions_errored_uniq")), 
    FunctionCall(None, "sumIfMerged", (Column(None, None, "sessions_errored_sum")) 
)))
``` 
This way when the query references `sessions_errored` the AST will be translated from a `Column(None, None, `sessions_errored`)` into that function call that runs a sum of two other function calls.